controllers/falcon/falconnodesensor_controller.go

```diff
@@ -155,7 +160,12 @@ func (r *FalconNodeSensorReconciler) handleConfigMaps(ctx context.Context, nodes
 		logger.Error(err, "error getting Configmap")
 		return nil, updated, err
 	} else {
-		err = r.Update(ctx, r.nodeSensorConfigmap(cmName, nodesensor, logger))
+		configmap, err := r.nodeSensorConfigmap(cmName, nodesensor)
+		if err != nil {
+			logger.Error(err, "Failed to format Configmap for update", "Configmap.Namespace", nodesensor.Namespace, "Configmap.Name", cmName)
+			return nil, updated, err
+		}
+		err = r.Update(ctx, configmap)
```

Comment:
Why not move the error message into `r.nodeSensorConfigmap`?

Reply:
Better to bring it back to the controller. The higher we bring it the better. One day we may need to create events or report the error into the `status:` subresource.